core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -326,7 +331,9 @@ private void initializeLocalWorker() throws IOException {
     List<WorkerInfo> infos;
     BlockMasterClient blockMasterClient = mBlockMasterClientPool.acquire();
     try {
-      infos = blockMasterClient.getWorkerInfoList();
+      infos = blockMasterClient.getWorkerInfoList(WorkerInfoOptions.defaults()
+          .setWorkerRange(WorkerRange.LIVE)
+          .setFieldRange(new HashSet<>(Arrays.asList(WorkerInfoField.ADDRESS))));
```

Comment:
Should we pull out `new HashSet<>(Arrays.asList(WorkerInfoField.ADDRESS))` as a constant?

Or, alternatively, we can save the option.

Reply:
It there a must to save it as a constant?
I would prefer to save it as a method variable in getOptions to make the cod cleaner.
Thanks!

Changed to 
```java
  Set<WorkerInfoField> fieldRange = new HashSet<>(Arrays.asList(WorkerInfoField.ADDRESS,
        WorkerInfoField.CAPACITY_BYTES, WorkerInfoField.CAPACITY_BYTES_ON_TIERS,
        WorkerInfoField.LAST_CONTACT_SEC, WorkerInfoField.USED_BYTES,
        WorkerInfoField.USED_BYTES_ON_TIERS));
  workerOptions.setFieldRange(fieldRange);
```